eds/plugins/github_provider.py

```diff
@@ -0,0 +1,143 @@
+import os
```

Comment:
as for the name of the builtin plugin modules, I was thinking this pattern:  

```
<interface name>_<plugin name>
```

so, in this case, "vcs_provider_github"

Reply:
👍 That works for me. I was planning to ask about this